src/pages/Main/components/TokensBlock/TokensBlock.tsx

```diff
@@ -0,0 +1,56 @@
+import { VFC } from 'react';
+import { ScreenXXS, ScreenXS, ScreenSM, ScreenLG, ScreenXL, ScreenXXL, ScreenMD } from '@app/styles/styled-components/screenWrappers';
+import { Tokens } from './Tokens';
+
+interface TokensBlockProps {
+  searchString?: string
+}
+
+export const TokensBlock: VFC<TokensBlockProps> = ({ searchString }) => {
+  return (
+    <>
+      <ScreenXXS>
+        <Tokens
+          itemsCount={6}
```

Comment:
Did you check how this works? We should use key prop for every recurring component on a single page.